src/components/CallView/shared/LocalMediaControls.vue

```diff
@@ -170,6 +170,8 @@ export default {
 			mouseover: false,
 			callAnalyzer: callAnalyzer,
 			qualityWarningInGracePeriodTimeout: null,
+			audioEnabledBeforeSpacebarKeydown: undefined,
+			spacebarKeyDown: false,
```

Comment:
`isPushToTalkActive` became `spacebarKeyDown` again :-(

Reply:
yes, because it can also be push to mute. That's why we decided to change it back.